conn.go

```diff
@@ -291,7 +291,7 @@ func (s *Session) dialWithoutObserver(ctx context.Context, host *HostInfo, cfg *
 		session:       s,
 		streams:       s.streamIDGenerator(cfg.ProtoVersion),
 		host:          host,
-		isSchemaV2:    true, // Try using "system.peers_v2" until proven otherwise
+		isSchemaV2:    false, // Do not try using "system.peers_v2" first, since ScyllaDB does not support it
```

Comment:
In the python version of it there was a check if it's scylla or not, don't we care about it here?

Reply:
Yes, I was lazy.
I can use isScyllaConn() I guess.

Reply:
Fixed.